library/vulnerabilities/path-traversal/unsafePathStart.ts

```diff
@@ -1,28 +1,38 @@
 import { isAbsolute, resolve } from "path";
 import { isWrapped } from "../../helpers/wrap";
 
-const linuxRootFolders = [
-  "/bin/",
-  "/boot/",
-  "/dev/",
-  "/etc/",
-  "/home/",
-  "/init/",
-  "/lib/",
-  "/media/",
-  "/mnt/",
-  "/opt/",
-  "/proc/",
-  "/root/",
-  "/run/",
-  "/sbin/",
-  "/srv/",
-  "/sys/",
-  "/tmp/",
-  "/usr/",
-  "/var/",
-];
-const dangerousPathStarts = [...linuxRootFolders, "c:/", "c:\\"];
+// All folder names in this array need to be lowercase
+const rootFolders = [
+  "bin",
+  "boot",
+  "dev",
+  "etc",
+  "home",
+  "init",
+  "lib",
+  "media",
+  "mnt",
+  "opt",
+  "proc",
+  "root",
+  "run",
+  "sbin",
+  "srv",
+  "sys",
+  "tmp",
+  "usr",
+  "var",
+  // macOS specific
+  "applications",
+  "cores",
+  "library",
+  "private",
+  "users",
+  "system",
+  "volumes",
+].map((f) => `/${f}/`);
```

Comment:
I think for search-ability, it's better to have the slashes present, also reduces the diff?

Reply:
Changed